scripts/install_postgresql.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -e
+
+_cmd() {
+    echo "$@"
+    "$@"
+}
+
+if [[ -z ${POSTGRESQL_VERSION} ]]; then
+	echo "POSTGRESQL_VERSION not set, skipping"
+	exit 0
+fi
+
+echo "Installing PostgreSQL ${POSTGRESQL_VERSION}"
+
+if [[ ! -z ${TRAVIS} ]]; then
+	case "${TRAVIS_OS_NAME}" in
+		linux)
+			_cmd sudo service postgresql stop
+			_cmd sudo service postgresql start ${POSTGRESQL_VERSION}
+			;;
+		osx)
+			DEFAULT_VERSION=9.6
+			# https://github.com/travis-ci/travis-ci/issues/1875
+			_cmd rm -rf /usr/local/var/postgres
+			if [[ ${POSTGRESQL_VERSION} != ${DEFAULT_VERSION} ]]; then
+				_cmd brew unlink postgresql
+				_cmd brew install postgresql@${POSTGRESQL_VERSION}
+				_cmd brew link --force postgresql@${POSTGRESQL_VERSION}
+			fi
+			_cmd initdb /usr/local/var/postgres
+			_cmd pg_ctl -D /usr/local/var/postgres start
+			sleep 1
+			_cmd createuser -s -p 5432 postgres
+			_cmd psql --version
```

Comment:
I think it would be a good idea to echo the psql version on Linux and appveyour, wouldn't it?

Reply:
Yes. Although it is tricky on Linux, since multiple `psql` versions co-exist. I'll check.